requirements.txt

```diff
@@ -3,18 +3,22 @@ attrs==19.3.0
 blinker==1.4
 certifi==2019.3.9
 chardet==3.0.4
-Click==7.1.2
+click==7.1.2
 colorama==0.4.1
+enochecker-cli==0.6.0
+enochecker-core==0.8.1
 Flask==1.0.3
```

Comment:
We could try if we can update some of these dependencies btw (post-PR accept)

Reply:
see #98